en/metrics.rst

```diff
@@ -150,6 +149,19 @@ Geospatial Libraries
   <script type="text/javascript" src="https://www.openhub.net/p/liblas/widgets/project_factoids_stats?format=js"></script>
   <script type="text/javascript" src="https://www.openhub.net/p/python-iris/widgets/project_factoids_stats?format=js"></script>
 
+Coordinate Reference System Libraries 
```

Comment:
@camerons if you want to revert the part that was pulled in, please create another pull request.